lib/hyku_addons/engine.rb

```diff
@@ -513,6 +519,9 @@ def self.dynamically_include_mixins
       # Insert at the end of the actor chain
       Hyrax::CurationConcern.actor_factory.use HykuAddons::Actors::TaskMaster::WorkActor
 
+      # Remove the Hyrax Orcid JSON Actor as we have our own
+      Hyrax::CurationConcern.actor_factory.middlewares.delete(::Hyrax::Actors::Orcid::JSONFieldsActor)
```

Comment:
1. I can't see **Hyrax::Actors::Orcid::JSONFieldsActor** in hyrax or hyku_addons. Instead i saw one with a different name here:  **HykuAddons::Actors::Orcid::JSONFieldsActor**
https://github.com/ubiquitypress/hyku_addons/blob/main/app/actors/hyku_addons/actors/json_fields_actor.rb

2. Do we really need to delete it if the version in hyrax-orcid is direct copy of the one in hyku_addons.
If we need to adjust that actor for hyku_addons do we need to always refer to hyrax-orcid